js/stdlib/ts/ALGO.ts

```diff
@@ -1808,9 +1808,11 @@ export const connectAccount = async (networkAccount: NetworkAccount): Promise<Ac
   const me_na = { networkAccount };
   const tokenAccepted = async (token:Token): Promise<boolean> => {
     debug(`tokenAccepted`, token);
-    // @ts-ignore
-    const r = await balanceOfM(me_na, token);
-    return ( r !== false );
```

Comment:
I don't like the duplication of the implementation here. I think you should make `balancesOfM` that can return `false`s and then `balancesOf` will do what `balanceOf` did an turn them into zeros.